docs/UserGuide.md

```diff
@@ -385,6 +368,15 @@ Example:
 
 **To update a lineup:**
 
+Format: `update L/LINEUP n/NEW_LINEUP_NAME`
+
+* Update the lineup name of lineup LINEUP to NEW_LINEUP_NAME
+
+Example:
+* `update L/Starting5 n/Worst5` will change lineup Starting5 to lineup Worst5
+
+**To update a schedule:**
+
```

Comment:
Hi I referring to page 10 of our UG, i think it should be this instead.

Format: update T/TEAM_NAME L/LINEUP_NAME n/NEW_LINEUP_NAME
Update the lineup name of lineup LINEUP_NAME to NEW_LINEUP_NAME

Example:
update T/Lokers L/worst5 n/best5 will change lineup worst5 in team Lokers to team Lakers

Reply:
Rights, edited accordingly